Src/AutoFixture/Kernel/BindingCommand.cs

```diff
@@ -86,6 +86,22 @@ public BindingCommand(Expression<Func<T, TProperty>> propertyPicker, Func<ISpeci
         }
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="BindingCommand{T, TProperty}"/> class with
+        /// the supplied property picker expression and a function that creates a value to be
+        /// assigned to that property or field.
+        /// </summary>
+        /// <param name="propertyPicker">An expression that identifies a property or field.</param>
+        /// <param name="valueCreator">
+        /// A function that creates a value that will be assigned to the property or field
+        /// identified by <paramref name="propertyPicker"/>.
+        /// </param>
+        public BindingCommand(Expression<Func<T, TProperty>> propertyPicker, Func<TProperty> valueCreator)

```

Comment:
This constructor overload adds no value, as we could use the following one instead:
```csharp
public BindingCommand(Expression<Func<T, TProperty>> propertyPicker, Func<ISpecimenContext, TProperty> valueCreator)
```

You can create a closure and wrap the passed factory here (like `_ => factory()`).
Likely, you created it as we have constructor that receives the value, however I'd vote to not introduce this one.